actions/vrfv2_helpers.go

```diff
@@ -0,0 +1,84 @@
+package actions
+
+//revive:disable:dot-imports
+import (
+	"fmt"
+	"math/big"
+
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+	uuid "github.com/satori/go.uuid"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/client"
+	"github.com/smartcontractkit/chainlink-testing-framework/contracts"
+)
+
+func DeployVrfv2Contracts(
+	linkTokenContract contracts.LinkToken,
+	contractDeployer contracts.ContractDeployer,
+	networks *blockchain.Networks,
+	linkEthFeedAddress string,
+) (contracts.VRFCoordinatorV2, contracts.VRFConsumerV2, contracts.BlockHashStore) {
+	bhs, err := contractDeployer.DeployBlockhashStore()
+	Expect(err).ShouldNot(HaveOccurred())
+	coordinator, err := contractDeployer.DeployVRFCoordinatorV2(linkTokenContract.Address(), bhs.Address(), linkEthFeedAddress)
+	Expect(err).ShouldNot(HaveOccurred())
+	consumer, err := contractDeployer.DeployVRFConsumerV2(linkTokenContract.Address(), coordinator.Address())
+	Expect(err).ShouldNot(HaveOccurred())
+	err = networks.Default.WaitForEvents()
+	Expect(err).ShouldNot(HaveOccurred())
+
+	return coordinator, consumer, bhs
+}
+
+func CreateVrfV2Jobs(
+	chainlinkNodes []client.Chainlink,
+	coordinator contracts.VRFCoordinatorV2,
+) ([]*client.Job, [][2]*big.Int) {
```

Comment:
Any way we can get a bit more type-safety around the proving keys? Not a big deal, just feels like an oddly unique data type to just throw out there.

Reply:
It is odd looking indeed. Will do.